internal/vul/scanner.go

```diff
@@ -111,11 +110,6 @@ func (s *imageScanner) Init(name, version string) {
 func (s *imageScanner) Stop() {
 	s.mx.RLock()
 	defer s.mx.RUnlock()
-
```

Comment:
I think we still need to close the store if we have one. since the closer interface is now moved to the store.
i.e

```go
	if s.store != nil {
		s.store.Close()
		s.store = nil
	}
```

Does this make sense?